chaoscenter/graphql/server/pkg/chaos_experiment/handler/handler.go

```diff
@@ -1513,7 +1513,7 @@ func (c *ChaosExperimentHandler) StopExperimentRuns(ctx context.Context, project
 		if len(experimentRunsID) == 0 && experiment.CronSyntax == "" {
 			return false, fmt.Errorf("no running or timeout experiments found")
 		}
-	} else if experimentRunID != nil && *experimentRunID != "" {
+	} else if *experimentRunID != "" {
```

Comment:
Hi @amityt :) Thanks to your review!
I removed `experimentRunID != nil` because we already check whether experimentRunID is nil above `if( experimentRunID == nil)` statement. So I thought at `else if` statement, possibility of nil is disappeared.

Reply:
Got it @jemlog! Have approved the PR 🚀